src/components/autocomplete/js/autocompleteController.js

```diff
@@ -614,7 +623,7 @@ function MdAutocompleteCtrl ($scope, $element, $mdUtil, $mdConstant, $mdTheming,
   /**
    * Clears the searchText value and selected item.
    */
-  function clearValue () {
+  function clearValue (clearText) {
```

Comment:
`clearText` here appears to be a boolean. Can we rename it to `shouldClearText` to be more specific? I originally thought it was a variable of what text to use as a replacement when clearing the old text.